kubeclient.gemspec

```diff
@@ -30,4 +30,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rest-client'
   spec.add_dependency 'recursive-open-struct', '~> 1.0.4'
   spec.add_dependency 'http', '~> 2.2.2'
+  spec.add_dependency 'json-streamer'
```

Comment:
it's the only one I'm aware of that provides the high-level utility of parsing whole objects out of streaming json bodies. I agree that it's a small project and preferable to use something with a bit of a community around it.

there is https://github.com/dgraham/json-stream which has more stars (if that's an indicator or anything). a bit lower level than `json-streamer`, it parses tokens from a streaming body. However in order to use it, we'd effectively have to duplicate the work that `json-streamer` takes care of for us (filtering objects by nesting level, key name, etc.).

I'm open to changing it to a more active/popular project. Do you have any suggestions?